BPOR/BPOR.Registration.Stream.Handler/src/BPOR.Registration.Stream.Handler/Handlers/StreamHandler.cs

```diff
@@ -127,6 +131,8 @@ private async Task ProcessModifyAsync(DynamoDBEvent.DynamodbStreamRecord record,
     {
         var identifiers = participantMapper.ExtractIdentifiers(record.Dynamodb.NewImage);
         var participant = await participantDbContext.GetParticipantByLinkedIdentifiers(identifiers)
+            .Include(x => x.ContactMethods)
```

Comment:
Won't be required here if added to `ForUpdate()`.

Reply:
moved inlcudes into forUpdate.